src/scripts/components/Tooltip.jsx

```diff
@@ -195,7 +171,19 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   openTooltip(tooltip) {
-    tooltip.timeline.play();
+    const { delay } = this.props;
+
+    // Convert delay to a number
+    // Takes care of multiple PropTypes
+    const delayDuration = Number.isInteger(delay) ? Number(delay) : 1000;
```

Comment:
In response to your previous comment:
```
So delay can either be a bool (which will default to that 1000) or an integer, to override that default amount.
```
If I pass in `3000` for `delay`, `Number.isInteger(3000)` is `true`, so `Number(3000)` is returned. My question was around if it's an integer, it is already a number, so wrapping in `Number()` seems unnecessary.

If you pass in `true` or `false` for `delay`, `Number.isInteger(<bool>)` is `false`, and you'll get `1000` for both.

Is the intention that you can pass `false` and the delay is 0? You can pass `true` and the delay is defaulted to `1000`? Or you can pass a number and the delay will be that? I think the logic is off here or I may be missing something...

Reply:
Ok, my bad. I should have read further. You're checking below if delay is `true` or `false`. Ignore comment above, except maybe that you might not need the `Number()` call.

Reply:
👍 Yep, you're right, we don't need `Number()` there.